package.json

```diff
@@ -6,17 +6,36 @@
   "author": "Calibre <hello@calibreapp.com>",
   "license": "ISC",
   "scripts": {
-    "test": "GITHUB_WORKSPACE=__tests__/test-images jest"
+    "test": "GITHUB_WORKSPACE=__tests__/test-images jest",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "format": "prettier --write **/*.ts",
+    "format-check": "prettier --check **/*.ts",
+    "lint": "eslint src/**/*.ts"
+  },
+  "prettier": {
+    "semi": false,
+    "singleQuote": true
   },
   "jest": {},
   "dependencies": {
     "@octokit/rest": "^16.17.0",
+    "@types/js-yaml": "^3.12.5",
     "glob": "^7.1.3",
+    "globby": "^11.0.1",
```

Comment:
What's `globby` for? @mikedijkstra

Reply:
Can be removed. I explored using it for better type support, but not required.

Reply:
Gotcha.

Reply:
Now that I think about it, it was for better promise support and not needing to use `util.promisify`, but again, not required!